third-party/libtm/libtm/include/TrackingCommon.h

```diff
@@ -62,9 +62,10 @@ namespace perc {
         Controller = 6,
         Rssi = 7,
         Velocimeter = 8,
-        Stereo = 9,
-        Pose = 10,
-        ControllerProperty = 11,
+        Mask = 9,
```

Comment:
This is API-breaking change for code compiled with previous versions. FYI

Reply:
Changed to non-breaking version